ElectronClient/app/gui/NoteText.jsx

```diff
@@ -454,14 +455,18 @@ class NoteTextComponent extends React.Component {
 		const stateNoteId = this.state.note ? this.state.note.id : null;
 		let noteId = null;
 		let note = null;
+		let newNote = null;
 		let loadingNewNote = true;
 		let parentFolder = null;
 		let noteTags = [];
 		let scrollPercent = 0;
 
 		if (props.newNote) {
-			note = Object.assign({}, props.newNote);
+			// assign new note and prevent body from being null
+			note = Object.assign(props.newNote, {body: ''});
```

Comment:
I don't quite understand the reason for this change. I believe it will directly modify the component prop, which should not be done. I think it should be `Object.assign({}, props.newNote, {body: ''});`

Reply:
The reason for that is because when using a template we assume that the note has a body (this can be changed), which I think is reasonable because it doesn't make sense for the note body to be null.
The reason for formatting the syntax that way is simply my own javascript ineptitude.